xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspCsvServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package de.adorsys.xs2a.adapter.registry;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.csv.CsvGenerator;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import de.adorsys.xs2a.adapter.registry.mapper.AspspMapper;
+import de.adorsys.xs2a.adapter.service.AspspCsvService;
+import de.adorsys.xs2a.adapter.service.model.Aspsp;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AspspCsvServiceImpl implements AspspCsvService {
+
+    private LuceneAspspRepository luceneAspspRepository;
+    private final AspspMapper aspspMapper = Mappers.getMapper(AspspMapper.class);
+
+    public AspspCsvServiceImpl(LuceneAspspRepository luceneAspspRepository) {
```

Comment:
If you agree with https://github.com/adorsys/xs2a-adapter/pull/182/files#r321748401, I would change the constructor parameter type as well to `AspspRepository`